### PR TITLE
htmldiff: initial ugly integration of htmldiff.js library

### PR DESCRIPTION
**Pull request by posborne**
This appears to be a better written library and I think a more promising
route moving forward.  There are some parts of the resultant output
that are not desirable right now, but I think some of that can be resolved
by some cleanup and styling changes.

@funkyeah This package is available on npm, but I wasn't sure how to integrate it so I just added it directly (as it was a single file and already written in coffeescript).  It isn't "perfect" but seems to work better -- we may need to customize.  Feedback on the right way to integrate appreciated.

### CODE REVIEW

File: client/views/history/history.coffee [hunk 1]
```diff
@@ -1,88 +1,9 @@
...
-
-plain2html = (plain) ->
-  for tag, c of _htmlHash
-    plain = plain.replace RegExp(c, 'g'), tag
-  plain
-
-
-# History comparison rendered function
-Template.compare.rendered = ->
+Template.compare.diff = ->
```
Comment (posborne) [inline client/views/history/history.coffee:1]: This probably has a reactive race condition that could be dealt with pretty easily

**Comment by posborne** [comment]
It works.  Its gettin' merged.  feedback still very welcome.

**Comment by funkyeah** [comment]
Checkout (
https://github.com/oortcloud/unofficial-meteor-faq#what-about-npm-node-modules)
for a decent overview of NPM use in meteor

the TL;DR is basically that you can
A) you can require/use NPM packages by creating a meteorite package (
https://atmosphere.meteor.com/wtf/package)
or
B) install a 3rd party package that allows the use of straight Npm.require
(in http://meteorhacks.com/complete-npm-integration-for-meteor.html)

I am down for going whatever direction on the diffing, I went away from
htmldiff mostly because its inactive and looked less polished than things
relying on diff match patch. For some of the reasons outlined in that email
I just sent you, the diff match patch route has its own challenges (not
really meant to work on tree structured documents like html), so you have
to workaround it, and I am not sure if its worth it.

Given that its inactive its probably legit to just add it to our project.
Where it lives depends on where you think it should live. I imagine either
where it is, in client/lib, or some other place where the lib files all go.
